drivers/adc/ad7124/ad7124_regs.c

```diff
@@ -1,3 +1,44 @@
+/***************************************************************************//**
+*   @file    AD7124-regs.c
+*   @brief   AD7124 register file.
+*   @devices AD7124-4, AD7124-8
+*
+********************************************************************************
+* Copyright 2015(c) Analog Devices, Inc.
```

Comment:
curios: why 2015?
why not something newer;
assuming it's a year range [because the driver was probably written in 2015], you could also do `2015-2019`

Reply:
done.